apps/web/src/styles/index.css

```
@@ -333,3 +333,195 @@
+  position: relative;
+  margin-bottom: 0.4rem;
+}
+
+.docs-prose ul > li::before {
+  content: "";
+  position: absolute;
+  left: -1.25rem;
+  top: 0.6em;
+  width: 0.375rem;
+  height: 0.375rem;
+  background-color: hsl(var(--primary) / 0.6);
```

Stylelint (17.7.0)

[error] Expected "0.6" to be "60%" (alpha-value-notation)

(alpha-value-notation)